src/ray/gcs/gcs_client/service_based_accessor.cc

```diff
@@ -753,12 +753,17 @@ void ServiceBasedNodeInfoAccessor::HandleNotification(const GcsNodeInfo &node_in
     // was alive and is now dead or resources have been updated.
     bool was_alive = (entry->second.state() == GcsNodeInfo::ALIVE);
     is_notif_new = was_alive && !is_alive;
-    // Once a node with a given ID has been removed, it should never be added
-    // again. If the entry was in the cache and the node was deleted, check
-    // that this new notification is not an insertion.
-    if (!was_alive) {
-      RAY_CHECK(!is_alive)
-          << "Notification for addition of a node that was already removed:" << node_id;
+    
+    // When a new node(node-B) registers with GCS, it subscribes to all node information.
```

Comment:
Please keep the original comment and add a supplement after it.

Reply:
The original comment was inconsistent with the current implementation, so I deleted it.

Reply:
How about this:

```
// Once a node with a given ID has been removed, it should never be added
// again. If the entry was in the cache and the node was deleted, we should check
// that this new notification is not an insertion.
// However, ...
// So we use `RAY_LOG` instead of `RAY_CHECK ` as a workaround.
```

Reply:
SGTM, i will fix it, thanks.